### PR TITLE
(Delegator) Go delegator migration

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

structure's nearly same as api
checked it running datafeed through local docker compose (baobab)

### TODO

- there's one test which is being skipped in sign test. (couldn't still get how to send signed raw tx through go klaytn lib), please help if anybody knows

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: go-delegator/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Go Delegator
+
+- Go implementation for delegator
+
+## TODOs
```
Comment (martinkersner) [inline go-delegator/README.md:5]: Is this still relevant?

File: go-delegator/function_/queries.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package function_
```
Comment (martinkersner) [inline go-delegator/function_/queries.go:1]: Is `function` a go reserved keyword? According to https://go.dev/ref/spec#Keywords it is not 🤔

File: go-delegator/go.mod [hunk 3]
```diff
@@ -0,0 +1,159 @@
+module bisonai.com/orakl/go-delegator
```
Comment (martinkersner) [inline go-delegator/go.mod:1]: I think we could call it `delegator` already know. It has no overlap with js delegator namespace.

```suggestion
module bisonai.com/orakl/delegator
```

File: go-delegator/main.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	}
+
+	err = app.Listen(":" + port)
+	if err != nil {
+		panic(err)
+	}
+
+}
+
+func SetRouter(_router fiber.Router) {
```
Comment (martinkersner) [inline go-delegator/main.go:54]: I am still confused by the `_` underscores you use from time to time. Is there a reason behind using it in this case? 🤔

**Review by martinkersner — APPROVED** [review]
LGTM! How about we try load testing with https://k6.io/? 😄

File: go-delegator/main.go (hunk 4 above)
Comment (nick-bisonai) [reply go-delegator/main.go:54]: sometimes I add underscored variable if it had to be redefined again. And then after modification and cleanup I forget and just leave the underscored ones. Thank you for pointing out, i'll fix it

File: go-delegator/README.md (hunk 1 above)
Comment (nick-bisonai) [reply go-delegator/README.md:5]: I'll update it 👍

File: go-delegator/function_/queries.go (hunk 2 above)
Comment (nick-bisonai) [reply go-delegator/function_/queries.go:1]: Oh I thought it was reserved, just like error in api
thank you for pointing out

**Pull request merged**